Package.swift

```diff
@@ -27,6 +27,7 @@ let package = Package(
         .Package(url: "https://github.com/IBM-Swift/Kitura.git", majorVersion: 1, minor: 3),
         .Package(url: "https://github.com/IBM-Swift/HeliumLogger.git", majorVersion: 1, minor: 3),
         .Package(url: "https://github.com/IBM-Swift/Kitura-MustacheTemplateEngine.git", majorVersion: 1, minor: 1),
-        .Package(url: "https://github.com/IBM-Swift/Kitura-StencilTemplateEngine.git", majorVersion: 1, minor: 3),
+        .Package(url: "https://github.com/IBM-Swift/Kitura-StencilTemplateEngine.git", majorVersion: 1, minor: 4),
+        .Package(url: "https://github.com/kylef/Stencil", majorVersion: 0, minor: 7)
```

Comment:
> IIRC SemVer allows for API-breaking changes on minor version change if the major version number is 0, like in your example.

Hmm, so in case of major version 0, increasing a minor version in the version of C can introduce a breaking API change. So you think that instead of having a version conflict immediately, it is better to proceed with fetching and compilation, until the compiler error?

Still, so what harm will arise from specifying only the major version?
```swift
.Package(url: "https://github.com/kylef/Stencil", majorVersion: 0)
```
Only when Kitura-TemplateEngine will switch to Stencil 1.0.0, a version conflict will happen, and it will force the author of Kitura-Sample to check the differences in the APIs of Stencil and to update the major version to 1.